snuba/web/views.py

```diff
@@ -283,7 +304,7 @@ def _trace_transaction(dataset: Dataset) -> None:
 
 @application.route("/query", methods=["GET", "POST"])
 @util.time_request("query")
-def unqualified_query_view(*, timer: Timer):
+def unqualified_query_view(*, timer: Timer) -> WerkResponse:
```

Comment:
redirect returns a werkzeug response instead of a Flask Response

Reply:
total nitpick, but `WerkzeugResponse` reads better `WerkResponse` to me

Reply:
ok